### ISSUE TITLE
[Feature Request] TAStudio's non-save state marker, marker.

### ISSUE THREAD

**Issue by Spikestuff** [TASEmulators/BizHawk#3892]
I only realized this whilst using Encore and then thought about it.
Basically a way to have a main marker that acts as a state (so what Hawk currently has) and then secondary markers to be like where something works, and you want to check around it and it goes back to that main marker.
This would also presumably reduce the filesize of tasproj files if used effectively by the TASer.

Like for an example on what I mean by "where something works" when I TAS Tekken I place markers on where something connects the opposing player, and I want to know that it can be there, and check around that frame instead of having that as a state, I can have that as an idea of "Yes, I can confirm that they get hit here, but I want to know what happens if I do this instead".

**Comment by Morilli** [comment]
Isn't this just branch + marker? Set a marker on the interesting frame, and save a branch on the inputs that work well. Then you could experiment and go back to the branch if you aren't successful.

**Comment by RetroEdit** [comment]
This feature request piqued my interest because I've been thinking about marker-related improvements for a while, and on a re-read I guess the feature request is for certain markers to not have savestates associated with them for cores with memory-intensive cores like Encore? (The current default having been for a while that all or most(?) savestates have attached savestates associated with them.)

I am wary of increasing the complexity of using markers, but this might fit well with some other features I want for markers.

**Comment by SuuperW** [comment]
This is implemented in PR #4774

**Comment by Spikestuff** [comment]
Nope, it's not resolved. Based on how that PR is written that's implying that I can disable it.
Whereas as I've written both at the same time.

Unless I'm misreading it then lol.

**Comment by SuuperW** [comment]
> I can disable it.

Isn't disabling states on individual markers exactly what you requested, though?

**Comment by Spikestuff** [comment]
No, both at the same time.

I personally like the markers that are saves, but I would like markers that aren't saves so I know when something lands, but I don't want it to remove my saving functionality.

**Comment by SuuperW** [comment]
I do not understand. What does "both at the same time mean", if not having some markers that have states while others do not?

**Comment by Spikestuff** [comment]
Did you make the PR based on being able to disable it outright or not?
Cause I don't want to disable it outright.

That's why I also wrote "Unless I'm misreading it then".

**Comment by SuuperW** [comment]
No, of course not.

I don't know if you know this, but you can actually test any PR by clicking on the "checks" tab then clicking on "artifacts". For my PR above, that will give you https://github.com/TASEmulators/BizHawk/suites/74656975031/artifacts/7706259139. If anything in the PR description is not clear, testing it can answer your questions.
